src/components/LayerInfosDialog/__snapshots__/LayerInfosDialog.test.js.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`LayerInfosDialog should match snapshot when Layer is defined 1`] = `null`;
```

Comment:
snapshot always empty currently. Better use a real layer for testing?

Reply:
that's the point. I want it returns null if there is no layer.

Reply:
Tests fixed

Reply:
Thanks :)